integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -53,11 +53,14 @@ public class ArgumentParser {
           Arrays.asList("transport.properties.", "transport.urlParams.", "transport.headers."));
   public static final String SPARK_CONF_CUSTOM_ENVIRONMENT_VARIABLES =
       "spark.openlineage.facets.custom_environment_variables";
+  public static final String SPARK_CONF_EVENT_PROCESSING_TIMEOUT_SECONDS =
+      "spark.openlineage.event.processing.timeout.seconds";
```

Comment:
please update changelog and readme. 
you can also prepare additional PR to update docs. [docs](https://github.com/OpenLineage/docs/blob/main/docs/integrations/spark/spark.md) and readme contain same information, copied into two locations. Sorry for inconvenience.